src/test/nodestore/DatabaseShard_test.cpp

```diff
@@ -1140,6 +1135,20 @@ class DatabaseShard_test : public TestBase
                 if (createShard(data, *db) < 0)
                     return;
             }
+
+            boost::filesystem::path path(shardDir.path());
+            path /= "1";
+
+            auto static const ripemd160Key =
+                ripemd160File((path / "nudb.key").string());
+            auto static const ripemd160Dat =
+                ripemd160File((path / "nudb.dat").string());
+
+            BEAST_EXPECT(
+                ripemd160File((path / "nudb.key").string()) == ripemd160Key);
+            BEAST_EXPECT(
+                ripemd160File((path / "nudb.dat").string()) == ripemd160Dat);
```

Comment:
I don't think these two checks are necessary here as we perform them at the end of the loop.

Reply:
Done